src/components/interaction-designer/block-editors/MaxDurationSecondsEditor.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div v-if="hasIvr" class="form-group block-max-duration-seconds">
+  <div v-if="hasIvr" class="form-group block-max-duration-seconds" :id="`${block.uuid}.config.ivr.maxDurationSeconds`">
```

Comment:
I propose we add `:ref="${block.uuid}/..."` too (same value as `:id`), for vuejs DOM manipulation best practice. What do you think @bzabos ?

Reply:
We are using the :id to scroll a particular dom element into view. I am not sure how :ref would add an advantage as we would need to access the dom from ref to `scrollIntoView`. 
Ex: 
```
const [el] = this.$refs[`${block.uuid}.config.ivr.maxDurationSeconds`];
if (el) {
  el.scrollIntoView({ behavior: "smooth", block: "center" });
}
```
@safydy-r  Please help me understand what advantages we will have with adding `:ref`

Reply:
@poojahpatil90 , I'm not an expert on `ref` usage yet. So this may give a summary: https://jai.me/blog/vuejs-ref-attribute (or this stackoverflow opinion https://stackoverflow.com/a/64750868/3256489)